### PR TITLE
Add support for resizing of Simplivity datastore

### PR DESCRIPTION
**Pull request by svt-hchaudhari**
Signed-off-by: Hemant Chaudhari <hchaudhari@hpe.com>

### Description
Add support for resizing of Simplivity datastore

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/datastores.py [hunk 1]
```diff
@@ -88,6 +88,11 @@
 print(f"{datastore}")
 print(f"{pp.pformat(datastore.data)} \n")
 
+# Resize the datastore to 1 GB
```
Comment (svt-kmcdaniel) [inline examples/datastores.py]: recommend getting the current size of the datastore and adding 1 GB to it instead.

File: tests/unit/resources/test_datastores.py [hunk 2]
```diff
@@ -160,6 +160,20 @@ def test_delete(self, mock_delete):
...
+    @mock.patch.object(Connection, "post")
+    @mock.patch.object(Connection, "get")
+    def test_resize(self, mock_get, mock_post):
+        resource_data = [{'id': '12345', 'name': 'ds1', 'size': 2048}]
+        mock_get.return_value = {datastores.DATA_FIELD: resource_data}
+        mock_post.return_value = None, [{'object_id': '12345'}]
+        datastore_data = {'name': 'name1', 'id': '12345', 'size': 1024}
+        datastore = self.datastores.get_by_data(datastore_data)
+        datastore.resize(2048)
+        data = {'size': 2048}
```
Comment (svt-kmcdaniel) [inline tests/unit/resources/test_datastores.py]: move data variable to right before the mock_post.assert_called_once_with, or embed the information in the parameters

**Review by svt-kmcdaniel — CHANGES_REQUESTED** [review]

File: examples/datastores.py (hunk 1 above)
Comment (svt-hchaudhari) [reply examples/datastores.py]: Updated the code

File: tests/unit/resources/test_datastores.py (hunk 2 above)
Comment (svt-hchaudhari) [reply tests/unit/resources/test_datastores.py]: updated the code

**Review by sijeesh — APPROVED** [review]
LGTM

**Pull request merged**